### PR TITLE
Ho aggiunto es1.2

### PR DESCRIPTION
**Pull request by LorenzoDomina**
Add es1.2

### CODE REVIEW

File: cpp/es1.2/es2_day1.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+
+int main() {
+    vector<int> v;
+    int n;
```
Comment (giannivalenti) [inline cpp/es1.2/es2_day1.cpp]: You could move this variable declaration inside the `do`-`while` loop, actually inside the `if (scelta == 1)` block: you don't need it anymore once you pushed it back into the vector.

**Review by giannivalenti — CHANGES_REQUESTED** [review]
I would suggest some changes:
- please format the code using an automatic formatter (with VSCode you can format the opened document with `Shift` + `Alt` + `F`),
- please alwasy use english in your code (`scelta` => `choice`) &mdash; this comment doesn't apply to user messages,
- please add some comments that help the reader to understand your code, you should put each comment in a separate line, ideally just before the code line it refers to.

File: cpp/es1.2/es2_day1.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        cin >> scelta;
+    
+        if(scelta == 1) {
+            cout << "inserisci un numero: ";
+            cin >> n;
+            v.push_back(n);
+        }
+    } while (scelta == 1);
+
+    if(!v.empty()) {
```
Comment (lucaotta) [inline cpp/es1.2/es2_day1.cpp]: Hai fatto bene a controllare la condizione di array vuoto!

**Review by giannivalenti — APPROVED** [review]

**Comment by giannivalenti** [comment]
@lucaotta , per me questa PR va bene. Ti lascio la parola.

**Review by lucaotta — APPROVED** [review]
@LorenzoDomina per me ok, vai di merge! :rocket:

**Pull request merged**